hack/loadtest/test-crp2.yaml

```diff
@@ -33,5 +41,5 @@ spec:
     rollingUpdate:
       maxUnavailable: 100%
       maxSurge: 25%
-      unavailablePeriodSeconds: 60
+      unavailablePeriodSeconds: 15
   revisionHistoryLimit: 15
```

Comment:
A nit: an empty line at the end of the file would be great. Just to satisfy GitHub linters.

Reply:
Safe for the other few files.